### ISSUE TITLE
GPT-1 到 ChatGPT 简介 | Measure Zero

### ISSUE THREAD

**Issue by Shiina18** [Shiina18/shiina18.github.io#171]
https://shiina18.github.io/machine%20learning/2022/12/24/gpt/ 

 总体时间线参考 这里. GPT-1~3 GPT-1 Our system works in two stages; first we train a transformer model on a very large amount of data in an unsupervised manner — using language modeling as a training signal — then we fine-tune this model on much smaller supervised datasets to help it solve specific tasks. We trained a 12-layer decoder-only transformer with masked self-attention heads (768 dimensional states and 12 attention heads). GPT 全称 generative pre-training, 就是预训练 + 微调. 时间顺序从前到后依次是, GPT-1, BERT, GPT-2.
